src/chocolatey.tests/infrastructure.app/nuget/NugetCommonSpecs.cs

```diff
@@ -175,14 +178,12 @@ public void Should_set_user_agent_string()
                 Context();
                 var source = "https://community.chocolatey.org/api/v2/";
                 _configuration.Sources = source;
-                _configuration.Information.ChocolateyProductVersion = "vNext";
 
                 _because();
 
-                // Change this when the NuGet version is updated.
-                var nugetClientVersion = "6.4.1";

```

Comment:
Why was the assertion on the NuGet.Client version removed here?

Reply:
I'll go back and check, but there's no reason not to have the explicit version check here I suppose?

Reply:
Do we consider this part of the test actually valuable // important to the actual test here, actually?

If the test is simply checking that NuGet _is_ appending its version number to the user agent, it likely doesn't matter all that much to check the exact one, no? Especially since it's the NuGet libraries that are appending this, so IMO the _value_ of the version here is a bit out of scope of this test.

If we want a specific test for "we are using vX.X.X of NuGet.Client libraries" I don't know if this is a good place for it?

Reply:
My two cents, this IMO is not the place to do an explicit nuget version assertion. If we want to do that, that should be its own test.

This test is to ensure that the user agent is being set with a format we expect, not the explicit values themself.

Reply:
The main issue that I was bringing up here was that prior to this change, there _was_ an explicit check for the NuGet.Client version number in the User Agent string.

If we are removing this, then I was curious to understand the thought process behind this change.

If this test isn't the correct place to make this assertion, then that is fine, however, it does bring up the question of the validity of the check, and whether having it being done elsewhere would still be a good idea.

I don't think removing assertions, without putting them somewhere else, is a good idea.

Reply:
> This test is to ensure that the user agent is being set with a format we expect, not the explicit values themself.

Yes, understood.  However, that assertion _was_ there previously, so we would be losing this assertion, if it isn't put somewhere else.

Reply:
I'm not really sure what the reason for having a unit test for this is.

For now I'll restore it but I don't really see the point here, myself. It just adds an extra place we have to update when we want to update the nuget libraries we're using, I think? I'm not sure I see the value in that.

Reply:
Just to be clear...

I wasn't in any way tied to having this NuGet Version check within this test.

The reason that I raised this was due to the change in the assertion within the test.  There was no discussion about this happening, and why it was being done (not in the commit messages, nor within the PR itself), so I felt it had to be brought up for discussion.

If we want to remove this assertion, and there is a good reason to do this, then by all means do this (perhaps as a follow up issue since it is not strictly part of this work).

Reply:
@vexx32 @AdmiringWorm happy for this to be marked as resolved as well, just didn't want to click the button until I knew my last message had been seen.

Reply:
I'll make a note to file a follow up issue for this after lunch, sure! We can discuss further there.